client/src/store/index.ts

```diff
@@ -1,15 +1,48 @@
 import { configureStore } from '@reduxjs/toolkit';
+import storage from 'redux-persist/lib/storage';
```

Comment:
so far it seems to be working well... a bit too well, perhaps. i'm trying to find a way to clear out local storage, but after clearing out the storage, the previous items still persist. i mostly want to clear out storage for 'logout'... but whenever i logout, the previous items come back and i remain logged in... so i only see the logout screen for half a second.. if any of this makes sense.

Reply:
after further testing, i think its because the global variables for token still have data... i'm currently researching how to clear server-side variables.

Reply:
okay, yeah. redux-persist is awesome. most of the issues i was dealing with revolved around clearing global variables on server side.